src/core/include/openvino/op/util/detection_output_base.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace op {
+namespace util {
+/// \brief DetectionOutputBase basic class for DetectionOutput v0 and v8
+class OPENVINO_API DetectionOutputBase : public Op {
+public:
+    struct AttributesBase {
+        int background_label_id = 0;
+        int top_k = -1;
+        bool variance_encoded_in_target = false;
+        std::vector<int> keep_top_k;
+        std::string code_type = std::string{"caffe.PriorBoxParameter.CORNER"};
```

Comment:
Do we still need to have `caffe` in strings?

Reply:
yes, this value is based on specification